ACE/ace/OS_NS_Thread.cpp

```diff
@@ -4806,6 +4807,18 @@ ACE_OS::unique_name (const void *object,
 }
 #endif
 
+#ifdef ACE_HAS_GETTID
+pid_t
+ACE_OS::thr_gettid ()
```

Comment:
Consider if this should be provided unconditionally, and if it should just fall back to tid on other platforms/versions.